metricbeat/module/mysql/supported-versions.yml

```diff
@@ -0,0 +1,8 @@
+variants:
+  - {'MYSQL_VARIANT': 'mysql',   'MYSQL_VERSION': '5.7.12'}
+  - {'MYSQL_VARIANT': 'mysql',   'MYSQL_VERSION': '8.0.13'}
+  - {'MYSQL_VARIANT': 'percona', 'MYSQL_VERSION': '5.7.24'}
+  - {'MYSQL_VARIANT': 'percona', 'MYSQL_VERSION': '8.0.13-4'}
+  - {'MYSQL_VARIANT': 'mariadb', 'MYSQL_VERSION': '10.2.23'}
+  - {'MYSQL_VARIANT': 'mariadb', 'MYSQL_VERSION': '10.3.14'}
+  - {'MYSQL_VARIANT': 'mariadb', 'MYSQL_VERSION': '10.4.4'}
```

Comment:
@mdelapenya when we agree on the format and location of this file I will open this for review.

Reply:
I have moved these files to the `_meta` directory of each module as in #14355.